### ISSUE TITLE
[4 Request] Custom Field Media support for docs,audios,videos

### ISSUE THREAD

**Issue by micker** [joomla/joomla-cms#36252]
### Steps to reproduce the issue
1 create a media field
2 try to upload an mp3

### Expected result
display a audio player with mp3

### Actual result
we can't upload an mp3
but we didn't have restriction media is more large than an image ?

**Comment by dgrammatiko** [comment]
It works for the other types, you need to add in the xml something like `type=“docs,audios,videos,images”` iirc

**Comment by micker** [comment]
i try to find option in field administration but nothing
you recommanded to override a core file ... ?
thanks for your reply @dgrammatiko

**Comment by dgrammatiko** [comment]
> i try to find option in field administration but nothing

This is controlled from the developer. If they choose to use only the images then ask them to change their code.
BTW is this about custom fields?

**Comment by micker** [comment]
is for display an audio file (mp3) with player
it can be so simple with a media field ?

**Comment by dgrammatiko** [comment]
Custom field Media afaik is not updated yet to support the other types

**Comment by micker** [comment]
dommage !

**Comment by dgrammatiko** [comment]
If this is your request here then change the title to `[Request] Custom Field Media support for docs,audios,videos`

@bembelimen this should be fixed in 4.1, it's an easy one: add one more field (as multiple select) for `type` and in the layout some code to switch between 4 different types (images,docs,audios,videos).

**Comment by LukasHH** [comment]
i have here a custom field for other media files. https://github.com/LukasHH/plg_field_file

**Comment by micker** [comment]
@LukasHH cool adding ! no way to propose it in core ? to much change ?

**Comment by LukasHH** [comment]
We have just discovered that this no longer works with the update to version 4.0.5.

The same applies to the Media field when it is in an XML form. The types specification is ignored.

Otherwise, it can be included in the core if desired.

**Comment by micker** [comment]
Ahh dommage i realy think that a media field need to be use for media not only for field lol

**Comment by LukasHH** [comment]
The media field is part of accessiblemedia. In 4.0, types (comma separated types of files for Media Manager) was added. I use the field for the custom field.

Since the specification of types is ignored, I will open an Issiu for it.

**Comment by LukasHH** [comment]
No Stop ignore my 2 last comments.
The error is in front of the screen.

I had installed the JCE editor and its media manager replaced the one from Joomla core. If I switch off the one used by the core, then everything works.

**Comment by dgrammatiko** [comment]
Yeah JCE is replacing both the media field and the media manager, which probably is not what most people should expect for Joomla 4...

**Comment by LukasHH** [comment]
Yes, exactly - when you install the JCE, the configuration of the JCE automatically sets it to use its file browser instead of the media manager from the core. You have to switch this off and then it works.
![image](https://user-images.githubusercontent.com/31522294/149621213-a95d2314-b61f-4a2b-aa6d-60eb2932ed43.png)

**Comment by dgrammatiko** [comment]
@LukasHH The core tinyMCE is in feature parity with JCE so may I ask why you installed JCE?

**Comment by LukasHH** [comment]
This was in a local test installation and had no particular reason. Yesterday I was approached by someone that it wasn't working for them. That's the reason we came up with it. Today I found out that the JCE setting is causing it.

**Comment by QuyTon** [comment]
Please test PR #45013.
